src/ReleaseHistory.md

```diff
@@ -5,6 +5,8 @@
 * BUGFIX: Fix `error ERR997.ExceptionLoadingPdb : '[filename]' was not evaluated because its PDB could not be loaded (E_PDB_NOT_FOUND).` when reading PE file built with `PDBPageSize:8192` or greater, by upgrading msdia140.dll from `14.27.28826.96` to `14.32.31326.0`. [685](https://github.com/microsoft/binskim/pull/685)
 * FEATURE: Add BA3031.EnableClangSafeStack, rename BA3030.UseCheckedFunctionsWithGcc to BA3030.UseGccCheckedFunctions [#663](https://github.com/microsoft/binskim/pull/663)
 * Bump Sarif.Sdk by updating submodule from [fc9a9df to 698adb6](https://github.com/microsoft/sarif-sdk/compare/fc9a9dfb865096b5aaa9fa3651854670940f7459...698adb6365a242c6bb75adde56e3bd4be39c21d7). [#674](https://github.com/microsoft/binskim/pull/674)
+* Introduce first performance rule BA6001.DisableIncrementalLinkingInReleaseBuilds [#667](https://github.com/microsoft/binskim/pull/667)
```

Comment:
>Introduce [](http://example.com/codeflow?start=2&length=9)

Fantastic changes, David. I've marked this approved but will wait until you have a chance to address my renaming/other minor suggestions before merging. Give me an ack when we're ready to go.

Reply:
Thanks for the review.  I'm just getting back from a short vacation so I'll need some time to look through the feedback in detail.  In the meantime I've merged the head of main into my branch so it is at least up to date in that regard.

Reply:
So, besides a minor point on switching to a different logging overload for failure cases (that specifies the severity explicitly), my feedback is mostly around renaming. If you're under the gun and want me to simply go in and try to address my concerns, let me know, it wouldn't take long.

Of course, if you want time/inclination to discuss naming conventions, let's go. :) It's surprising how much thinking has to go into this topic.

Reply:
There is no particular hurry on my side.  I would like to get to it today it will just be a few hours because I have other catching up to do.

Naming things is always a challenge :)

Reply:
Ok I think I addressed all of your feedback.  The one exception is whether there should be more code sharing for the "CanAnalyzePE" logic in the performance rules.  We can discuss that one further and decide if further changes are appropriate.  Thanks.

Reply:
OK, two small nits, update the rules metadata and expand the rule ids in the release notes to meet our standard, encapsulated in back-ticks, both the opaque and readable rule names are explicit. Once this is done, we'll merge.

We can go tinker in future with various things as required, based on testing, output review, etc.